src/main/java/project/Main.java

```diff
@@ -1,12 +1,31 @@
 package project;
 
+import de.gurkenlabs.litiengine.Game;
+import de.gurkenlabs.litiengine.GameListener;
+
 public class Main {
 
     /**
      *
      * @param args
      */
     public static void main(String[] args) {
+        Game.addGameListener(new GameListener() {
+            @Override
+            public void initialized(String... args) {
+                // do sth when game is initialized
+            }
+            @Override
+            public void started() {
+                // do sth when game started
+            }
 
+            @Override
+            public void terminated() {
+                // do sth when game terminated
+            }
+        });
```

Comment:
This should be implemented in an own class, maybe something like logic.GameState